com.unity.cinemachine/Samples~/3D Samples/Impulse/Impulse Wave.unity

```diff
@@ -38,7 +38,7 @@ RenderSettings:
   m_ReflectionIntensity: 1
```

Comment:
This object:
![image](https://user-images.githubusercontent.com/6424658/215223331-f89b42f7-3c69-486b-a813-a944fc45e6bc.png)

![image](https://user-images.githubusercontent.com/6424658/215223200-df7f5251-8238-463e-a0f6-2c46dfa995c3.png)

Which gives this when old input system is disabled:

![image](https://user-images.githubusercontent.com/6424658/215223254-c670328f-e005-4a26-8fc1-74b223409d30.png)

Is there a way to do this in a way that works out of the box with both old and new input systems?

Reply:
Fixed it in PR (https://github.com/Unity-Technologies/com.unity.cinemachine/pull/758) that targets this PR.